ci/fireci/fireci/commands.py

```diff
@@ -61,3 +63,38 @@ def smoke_tests(app_build_variant, test_apps_dir):
             os.path.join(cwd, 'build', 'm2repository')),
         workdir=location,
     )
+
+
+@click.argument('issue_number', required=True)
+@click.argument('repo_name', required=True)
+@click.argument('auth_token', required=True)
```

Comment:
Consider making all of the above `option()`s instead of arguments, to make the cli invocation more readable.
Additionally you can set `envvar` on each of the following options to make them optional:
issue_number: `envvar="PULL_NUMBER"`
repo_name: `envvar="REPO_NAME"`